lib/template/kit.spec

```diff
@@ -43,6 +43,9 @@ thus it should never be installed on end users' systems.
 export NO_BRP_CHECK_BYTECODE_VERSION=true
 install -d -m 0755 %{buildroot}%{_datadir}/tetra/
 tar -C %{buildroot}%{_datadir}/tetra --strip-components=1 -xJf %{S:0}
+%if 0%{?rhel} >= 8
+sed -i '1s/python\b/python2/' %{buildroot}%{_datadir}/tetra/apache-ant-1.9.7/bin/runant.py
```

Comment:
How about using the `pathfix.py` approach recommended by Fedora instead (on any Python file, wherever it is under the `tetra` directory? Would that work for RHEL as well?

https://fedoraproject.org/wiki/Changes/Make_ambiguous_python_shebangs_error

Reply:
This has been some time. I will see when I can catch up on the background again. Let me check if that also works.

Reply:
@moio , I finally did some testing on this.

It would require if statements for the `BuildRequires: %{_bindir}/pathfix.py` as well as for the logic in the `%install` section.
Also the file based `BuildRequires` needs entries in the OBS project config (unless additional conditions are added for a Fedora/RHEL8 differentiation)

So I would say that my original suggestion is the most generic approach. Maybe we can even remove the original `%if` statement on line L46.

Reply:
What I do not like about this solution is that every time the bundled ant is bumped we will need to remember to change the shebang and the `sed` line above.

Could we maybe bundle pathfix.py itself, add it as `Source:` of the `kit` and run the recommended line in the Fedora wiki, that works recursively and would fix any such bundled script, belonging to ant or not?

Reply:
Now I see where you are coming from.
Can you please pick one option and I will get it sorted:
1. How about some "clever" "replace all python scripts" line in the SPEC? Like a micro-version of pathfix.
2. Bundle pathfix as you say.

My thought is to keep complexity, code and readability as human friendly as possible (= encourages contributions).

Reply:
I am with you in keeping the complexity low and I trust you with whatever solution you think is best.